main/gatewayBT.cpp

```diff
@@ -1423,7 +1456,7 @@ void process_bledata(JsonObject& BLEdata) {
               BTConfig.intervalActiveScan = MinTimeBtwScan;
               BTConfig.scanDuration = MinScanDuration;
               Log.notice(F("Active and continuous scanning required, parameters adapted" CR));
-              stateBTMeasures(false);
+              // stateBTMeasures(false); // Disabling as it casues a segfault
```

Comment:
I suggest to increase the stack size of the task: https://github.com/1technophile/OpenMQTTGateway/blob/ddb9cc67cc8629a3eacc90ab9b0d82fa40bc90ce/main/gatewayBT.cpp#L873
and see if you still get the issue